Forms.html

```diff
@@ -0,0 +1,86 @@
+    <!--<link rel="stylesheet" type="text/css" href="style.css"> --> 
+        </head>
+        <body>
+    <form>
+        <fieldset>
+            <legend>Basic User Information:</legend>
+        <label for="fullname">Full name:</label>
+        <input type="text" id="fullname" name="fullname" required><br>
+        <label for="email">Email:</label>
+        <input type="email" id="email" name="email" required><br>
+        <label for="password">Password:</label>
+        <input type="password" id="password" name="password" required>
+        <label for="confirm_password">Confirm Password:</label>
+        <input type="password" id="confirm_password" name="confirm_password" required><br>
+        <label >Gender:</label><br>
+        <input type="radio" id="male" name="Gen" value="Male">
+        <label for="male">Male</label>
+        <input type="radio" id="female" name="Gen" value="Female">
+        <label for="female">Female</label>
+        <input type="radio" id="other" name="Gen" value="Other">
+        <label for="other">Other</label><br>
+        <label for="date_of_birth">Date of Birth:</label>
+        <input type="date" id="date_of_birth" name="date_of_birth">
+    </fieldset>
+    <fieldset>
+        <legend>Address Information:</legend>
+    <label for="address">Street Address:</label>
+    <textarea id="address" name="address"></textarea><br>
+    <label for="country">Country:</label>
+    <select id="country" name="country">
+    <option value="USA">USA</option>
+    <option value="Scotland">Scotland</option>
+    <option value="Moldova">Moldova</option>
+    <option value="Ucraine">Ucraine</option>
+    <option value="Sweeden">Sweeden</option>
+    </select><br>
+    <label for="city">City:</label>
+    <input type="text" id="city" name="city" > <br>
+    <label for="postal_code">Postal Code:</label>
+    <input type="number" id="postal_code" name="postal_code">
+    </fieldset>
+    <fieldset>
+        <legend>Preferences</legend>
+        <label>Preferred Contact Method</label>
+        <input type="checkbox" id="e-mail" name="contact_method" value="Email">
+        <label for="e-mail">Email</label>
+        <input type="checkbox" id="phone" name="contact_method" value="Phone">
+        <label for="phone">Phone</label>
+        <input type="checkbox" id="posmail" name="contact_method" value="Postal-Mail">
+        <label for="postal-mail">Postal Mail</label><br>
+        <label for="color">Favorite color:</label>
+        <input type="color" id="color" name="color">
+    </fieldset>
+    <fieldset>
+        <legend>Additional Details:</legend>
+        <label for="details">How did you hear about us?</label>
+        <select id="details" name="details">
+            <option value="internet">Internet</option>
+            <option value="friend">Friend</option>
+            <option value="advertisement">Advertisement</option>
+            <option value="magazine">Magazine</option>
+        </select><Br>
```

Comment:
don't use capital letters in tag names

Reply:
Okay, it's was just a misprint, i didn't observe
![image](https://github.com/DirCd/Tekwill-Begin/assets/141740084/9082999d-b30e-4846-bd8d-f111b880c4f5)